code/modules/integrated_electronics/subtypes/converters.dm

```diff
@@ -67,6 +67,40 @@
 	push_data()
 	activate_pin(2)
 
+/obj/item/integrated_circuit/converter/refcode
+	name = "reference encoder"
+	desc = "This circuit can convert a reference in encoded string,which can be transferred throught EPV2 circuit"
```

Comment:
`This circuit can encode a reference into a string, which can then be read by an EPV2 circuit.`